security-analysis/security-analysis-api/src/main/java/com/powsybl/security/action/GenerationRedispatchAction.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.action;
+
+import java.util.Objects;
+
+/**
+ * An action increasing or decreasing the active power generation of a generator.
+ *
+ * @author Hadrien Godard <hadrien.godard@artelys.com>
+ */
+public class GenerationRedispatchAction extends AbstractAction {
+
+    public static final String NAME = "GENERATION_REDISPATCH";
+
+    private final String generatorId;
+    private final boolean increasing;
```

Comment:
The boolean `increasing` is not needed as you have the `value`. What we should know is that if  `value` is the expected value for `targetP` or if `value` represents a relative variation. So maybe we have to add a boolean that explicits that ('delta' ?).